docs/cloud/agent/fargate.md

```diff
@@ -201,3 +201,28 @@ $ prefect agent start fargate
 :::warning Outbound Traffic
 If you encounter issues with Fargate raising errors in cases of client timeouts or inability to pull containers then you may need to adjust your `networkConfiguration`. Visit [this discussion thread](https://github.com/aws/amazon-ecs-agent/issues/1128#issuecomment-351545461) for more information on configuring AWS security groups.
 :::
+
+#### Prefect CLI Using Kwargs
+
+All configuration options for the Fargate Agent can also be provided to the `prefect agent start fargate` CLI command. They must match the camel casing used by boto3.
+
+```bash
+$ export AWS_ACCESS_KEY_ID=...
+$ export AWS_SECRET_ACCESS_KEY=...
+$ export REGION_NAME=us-east-1
+
+$ prefect agent start fargate cpu=256 memory=512 networkConfiguration="{'awsvpcConfiguration': {'assignPublicIp': 'ENABLED', 'subnets': ['my_subnet_id'], 'securityGroups': []}}"
```

Comment:
can these also be provided with `--cpu=256` flags?

Reply:
Yes they can. Do you think I should add a not about that? It's parsed in both cases depending on user preference `--cpu` == `cpu`

Reply:
I'll add a note